providers/dns/autodns/autodns.toml

```diff
@@ -0,0 +1,18 @@
+Name = "Autodns"
+Description = ''''''
+URL = "https://www.internetx.com/domains/autodns/"
+Code = "autodns"
+Since = "v3.1.0"
+
+Example = ''''''
+
+[Configuration]
+  [Configuration.Credentials]
+    AUTODNS_API_USER = "Username"
+    AUTODNS_API_PASSWORD = "User Password"
+    AUTODNS_ENDPOINT = "API endpoint URL"
+    AUTODNS_CONTEXT = "API context (4 for production, 1 for testing. Defaults to 4)"
+    AUTODNS_TTL = "The TTL of the TXT record used for the DNS challenge"
```

Comment:
could you follow the same pattern as other providers:
https://github.com/go-acme/lego/blob/a5a29187fea9f555ea43fae16ca65066c452d330/providers/dns/auroradns/auroradns.toml#L9-L17

Reply:
Done.